go/api/go_api/auth.py

```diff
@@ -47,11 +52,76 @@ def requestAvatarId(self, credentials):
         raise error.UnauthorizedLogin()
 
 
+class GoAuthBouncerCredentialFactory(BasicCredentialFactory):
+    implements(ICredentialFactory)
```

Comment:
Rather use the `@implementer(ICredentialFactory)` decorator. It's more readable and less magic. :-)